docker/cpu/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
 MAINTAINER caffe-maint@googlegroups.com
```

Comment:
`MAINTAINER` is being [deprecated](https://github.com/docker/docker/pull/25466) in Docker 1.13, you should use a label instead, might as well change it now.
You can use something like [this](https://github.com/NVIDIA/nvidia-docker/blob/ef3e6055059b515e8e3ee0801848bbad8b5efde9/centos-7/cuda/7.5/devel/Dockerfile#L2) instead.